Automattic/jetpack
# WordPress.com Toolbar module - remove sidebars for my sites and user info.

Fixes #

related to pe7F0s-1Bv-p2#comment-2050 and response pe7F0s-1Bv-p2#comment-2055

## Proposed changes:


* Removes the slide out menus for the "my sites" and "user info" sections of the masterbar module. This is accomplished by removing all menu items that are children of the `blog` and `user-info` menu ids.

Before:
![masterbar-before](https://github.com/Automattic/jetpack/assets/28742426/4930bb02-88fe-4bcf-b93c-3bba9b1a78e0)

After:
![masterbar-after](https://github.com/Automattic/jetpack/assets/28742426/4da5c335-f7e5-47d0-b408-092cf12794bb)

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* On a self hosted and jetpack connected site enable the "WordPress.com toolbar" setting. this can be found searching jetpack settings, or within the modules list:
<img width="1111" alt="Screenshot 2024-01-24 at 11 30 29 AM" src="https://github.com/Automattic/jetpack/assets/28742426/394eebec-21a3-4591-a06f-b914d293725e">
<img width="804" alt="Screenshot 2024-01-24 at 11 45 25 AM" src="https://github.com/Automattic/jetpack/assets/28742426/28e5fe52-2ca4-4d6a-80a1-a057cd5ac878">

* Enable this branch using the jetpack beta plugin. (instructions in github-actions comment below)
* Verify the "my sites" and "user info" sections of the masterbar redirect to WordPress.com when clicked, instead of opening the sidebars as shown in the gifs in the above description.
* Smoke test these changes on simple and WoA. There should be no changes in functionality here.
